private/Producers/FbxProducer/FbxProducerImpl.cpp

```diff
@@ -59,9 +59,14 @@ cd::Vec3f ConvertFbxScale(const fbxsdk::FbxVector4& scale)
 	return cd::Vec3f(static_cast<float>(scale[0]), static_cast<float>(scale[1]), static_cast<float>(scale[2]));
 }
 
-cd::Transform ConvertFbxTransform(const fbxsdk::FbxVector4& position, const fbxsdk::FbxQuaternion& rotation, const fbxsdk::FbxVector4& scale)
+cd::Transform ConvertFbxTransform(fbxsdk::FbxAMatrix Matrix)
```

Comment:
```suggestion
cd::Transform ConvertFbxTransform(const fbxsdk::FbxAMatrix& Matrix)
```
Avoid copy when passing matrix parameter.